packages/edit-site/src/components/header-edit-mode/tools-more-menu-group/index.js

```diff
@@ -14,7 +9,7 @@ const { Fill: ToolsMoreMenuGroup, Slot } = createSlotFill(
 
 ToolsMoreMenuGroup.Slot = ( { fillProps } ) => (
 	<Slot fillProps={ fillProps }>
-		{ ( fills ) => ! isEmpty( fills ) && fills }
+		{ ( fills ) => fills && fills.length > 0 }
```

Comment:
`fills` are always an array:

https://github.com/WordPress/gutenberg/blob/b4887bfaf366425605f40c0b60b03d933b12d21b/packages/components/src/slot-fill/slot.js#L65

I can actually remove the `fills &&` check, but decided to keep it just in case.